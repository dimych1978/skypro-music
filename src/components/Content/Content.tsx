import styles from './Content.module.css';
import Playlist from '../PlayList/Playlist';

const Content = () => {
  return (
    <div className={`${styles.centerblockContent} playlist-content`}>
      <div className={`${styles.contentTitle} playlist-title`}>
        <div className={`${styles.playlistTitleCol} col01`}>Трек</div>
        <div className={`${styles.playlistTitleCol} col02`}>Исполнитель</div>
        <div className={`${styles.playlistTitleCol} col03`}>Альбом</div>
        <div className={`${styles.playlistTitleCol} col04`}>
          <svg className={styles.playlistTitleSvg}>
            <use xlinkHref='/img/icon/sprite.svg#icon-watch'></use>
          </svg>
        </div>
      </div>
      <Playlist />
    </div>
  );
};

export default Content;
