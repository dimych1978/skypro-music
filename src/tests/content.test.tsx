import Content from '@/components/Content/Content';
import ReduxProvider from '@/store/ReduxProvider';
import { render } from '@testing-library/react';

it('renders correctly', () => {
  const tree = render(
    <ReduxProvider>
      <Content />
    </ReduxProvider>
  );
  expect(tree).toMatchSnapshot();
});
