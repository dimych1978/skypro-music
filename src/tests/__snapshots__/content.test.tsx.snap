// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="centerblockContent playlist-content"
      >
        <div
          class="contentTitle playlist-title"
        >
          <div
            class="playlistTitleCol col01"
          >
            Трек
          </div>
          <div
            class="playlistTitleCol col02"
          >
            Исполнитель
          </div>
          <div
            class="playlistTitleCol col03"
          >
            Альбом
          </div>
          <div
            class="playlistTitleCol col04"
          >
            <svg
              class="playlistTitleSvg"
            >
              <use
                xlink:href="/img/icon/sprite.svg#icon-watch"
              />
            </svg>
          </div>
        </div>
        <div
          class="contentPlaylist playlist"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="centerblockContent playlist-content"
    >
      <div
        class="contentTitle playlist-title"
      >
        <div
          class="playlistTitleCol col01"
        >
          Трек
        </div>
        <div
          class="playlistTitleCol col02"
        >
          Исполнитель
        </div>
        <div
          class="playlistTitleCol col03"
        >
          Альбом
        </div>
        <div
          class="playlistTitleCol col04"
        >
          <svg
            class="playlistTitleSvg"
          >
            <use
              xlink:href="/img/icon/sprite.svg#icon-watch"
            />
          </svg>
        </div>
      </div>
      <div
        class="contentPlaylist playlist"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
